analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/AnalysisVisitor.java

```diff
@@ -239,11 +239,11 @@ public List<Description> visit(JavadocComment n, Analyzer arg) {
     StringBuilder returns = new StringBuilder();
     Map<String, String> commentParams = new LinkedHashMap<>();
     Map<String, String> commentTypeParams = new LinkedHashMap<>();
-    // This regex splits the sentence into 2 if it ends with a . is followed by empty space.
-    String[] sentences = CommentHelperMethods.extractSummary(n).split("\\.\\s+", 2);
-    // Give back the . that is removed by the split.
-    String summary = (sentences.length > 0) ? sentences[0]+"." : null;
-    String remarks = (sentences.length > 1) ? sentences[1] : null;
+    // This regex splits the sentence into 2 at the first dot(.) it finds.
+    String[] sentences = CommentHelperMethods.extractSummary(n).split("\\.", 2);
```

Comment:
It may not be possible to drop the "...followed by empty space" feature here if we want to stay compatible with the Javadoc tool. The examples from the [link from earlier](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#firstsentence) show why – here, the expected first sentence is `This is a simulation of Prof.` (since the dot is followed by a space):

```java
/**
* This is a simulation of Prof. Knuth's MIX computer.
*/
```

Here, it is `This is a simulation of Prof.&nbsp;Knuth's MIX computer.` since the dot is not followed by a space:

```java
/**
* This is a simulation of Prof.&nbsp;Knuth's MIX computer.
*/
```

One way to split things on dot-and-space without consuming the dot-and-space is to use a zero-width lookahead or lookbehind, something like this: `(?<=\\.\s)`.

It would perhaps be nice to put a test case as well to show that this works like in the linked article.